src/components/products/RelatedProducts.tsx

```
@@ -98,79 +106,83 @@ const ProductCard = ({ product }: { product: Product }) => {
   return (
     <>
       <div className="flex justify-start items-center gap-4 mb-8 lg:mb-[4rem]">
-
         <div className="w-[12px] lg:w-[20px] h-[30px] lg:h-[40px] rounded bg-[#375B42]"></div>
-        <h2 className="text-base font-semibold text-[#375B42]">{t('buttons.relatedItems')}</h2>
+        <h2 className="text-base font-semibold text-[#375B42]">{t("buttons.relatedItems")}</h2>
       </div>
       <div className="grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4 gap-4 md:gap-6">
         {products.map((product) => (
-          <ProductCard key={product.id} product={product} />
+            <ProductCard product={product} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)